core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneOptimized.java

```diff
@@ -436,19 +407,18 @@ public CanonicalGene getNonAmbiguousGene(String geneId) {
      * @return a gene that can be non-ambiguously determined, or null if cannot.
      */
     public CanonicalGene getNonAmbiguousGene(String geneId, String chr) {
```

Comment:
Looks like chr argument can be dropped here.

Reply:
good catch! will remove chr.

Reply:
chr param removed.